tests/playwright-test/exit-code.spec.ts

```diff
@@ -36,8 +36,32 @@ test('should collect stdio', async ({ runInlineTest }) => {
   expect(exitCode).toBe(0);
   const testResult = report.suites[0].specs[0].tests[0].results[0];
   const { stdout, stderr } = testResult;
-  expect(stdout).toEqual([{ text: 'stdout text' }, { buffer: Buffer.from('stdout buffer').toString('base64') }]);
-  expect(stderr).toEqual([{ text: 'stderr text' }, { buffer: Buffer.from('stderr buffer').toString('base64') }]);
+  expect(stdout.map(e => e.text).join('')).toEqual('stdout textstdout buffer');
```

Comment:
note: platforms chunk/flush differently. We concatenate to have a good expectation which we can compare.